### ISSUE TITLE
[Quest][Classic] Mission Bug "Homeward Bound"

### ISSUE THREAD

**Issue by KingRedpass** [dalaranwow/dalaran-wow#4656]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

Quise hacer la misión de Pao'ka Swiftmountain en la zona de Thousand Needles, llamada "Homeward Bound". Que se basa en escoltar a Pao'ka Swiftmountain. 
Aca les dejo un link de la misión http://www.wowhead.com/quest=4770/homeward-bound  

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Sucede en Thousand Needles, mas específicamente en Highperch (en el link anterior se puede ver el mapa). Ocurre luego de un rato de dar aceptar a la misión, Pao'ka Swiftmountain te sigue algunos metros y luego desaparece sin mas, al igual que la misión. Luego de un rato Pao'ka Swiftmountain vuelve a aparecer en su lugar original, pero sin la misión, no se puede hacer otra vez ni entregar.  

**Expected behaviour**: Tell us what should happen instead
Lo que debería suceder es que Pao'ka Swiftmountain camine hasta llegar fuera de Highperch, en el camino yo tendría que defenderlo de los Highperch Wyvern o Patriarch para lograr llevarlo con vida hasta su destino.

**Steps to reproduce the problem**:

1. Ir a Highperch y aceptar la misión de Pao'ka Swiftmountain
2. Seguirlo
3. Ver que luego de unos metros él desaparece

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

Ingame ID of Pao'ka Swiftmountain creature / NPC is 10427.

**Include Screenshots from the issue if necessary**

![wowscrnshot_040317_203143](https://cloud.githubusercontent.com/assets/26886452/24638022/c4ef84b4-18bb-11e7-84fb-28ed256641f5.jpg)

**Comment by TeridanDW** [comment]
Hello and thanks for the report @KingRedpass 
Translation of the report thanks to google.


I wanted to do the mission of Pao'ka Swiftmountain in the area of ​​Thousand Needles, called "Homeward Bound". It is based on escorting Pao'ka Swiftmountain.

I leave you a link of the mission http://www.wowhead.com/quest=4770/homeward-bound

Current behaviour: Tell us what happens in detail (Where, When, What, Why, Who)
It happens in Thousand Needles, more specifically in Highperch (in the previous link you can see the map). It happens after a while of accepting the mission, Pao'ka Swiftmountain follows you a few meters and then disappears without more, like the mission. After a while Pao'ka Swiftmountain reappears in its original place, but without the mission, it can not be done again nor delivered.

Expected behaviour: Tell us what should happen instead.
What should happen is that Pao'ka Swiftmountain walks to get out of Highperch, along the way I would have to defend it from the Highperch Wyvern or Patriarch to get him alive to his destination.

Steps to reproduce the problem:
1. Go to Highperch and accept the mission of Pao'ka Swiftmountain
2. Follow it
3. See that after a few meters he disappears

Include proofs for this behaviour

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

Ingame ID of Pao'ka Swiftmountain creature / NPC is 10427.

Include Screenshots from the issue if necessary

**Comment by TeridanDW** [comment]
Regarding the issue (tested on ptr) - quest works, creature starts and ends it's path where he should, however there are 2 issues that i notice:
 - when player accepts the quest, he gets the credit immediately instead of the end of the path.
Note: Player gets the credit at the end of the path too, so only the first credit should be removed (at event start)

 - on quest start npc http://wotlk.openwow.com/npc=10427 attacks nearby http://wotlk.openwow.com/npc=4107 which is wrong, he should be set to deffensive on event start.

**Comment by KingRedpass** [comment]
Pero en mi caso me desapareció, no se que fue lo que paso, pero cuando mire atrás Pao'ka Swiftmountain ya no estaba y la misión había desaparecido de la guía de misiones (porque yo quería cancelarla y volverla a empezar, para ver si se solucionaba). Por lo que volví con Pao'ka Swiftmountain, a ver si la tenia otra vez, y no estaba, como si nunca hubiera estado esa misión. Saludos

**Comment by KingRedpass** [comment]
But in my case I disappeared, I do not know what happened, but when I look back Pao'ka Swiftmountain was gone and the mission had disappeared from the mission guide (because I wanted to cancel it and start it again, to see if it Solved). So I came back with Pao'ka Swiftmountain, to see if I had it again, and it was not, as if it had never been that mission. regards

**Comment by KingRedpass** [comment]
But in my case I disappeared, but there is nothing to do, but when I look back Pao'ka Swiftmountain was gone and the mission had disappeared from the mission guide (because I want to cancel it and start over, to see if it was solved). So I came back with Pao'ka Swiftmountain, see if I had it again, and it was not, as if the state never exists that mission. regards @TeridanDW

**Comment by Rushor** [comment]
the autocompletetion was fixed in https://github.com/dalaranwow/dalaran-wow/issues/5001
